usaspending_api/common/query_with_filters.py

```diff
@@ -390,10 +390,17 @@ class _DisasterEmergencyFundCodes(_Filter):
     @classmethod
     def generate_elasticsearch_query(cls, filter_values: List[str], query_type: _QueryType) -> ES_Q:
         def_codes_query = []
+        amounts_query = [
```

Comment:
Instead of adding a nonzero check to the DEF Codes filter can you use the `_NonzeroFields` filter below? 

Also, if must_not match works for nonzero and you have the time can you also update that filter since that is must cleaner than all of the extra ranges.